### PR TITLE
parallel testing

### PR DESCRIPTION
**Pull request by WilliamBergamin**
### Summary

By including the `--parallel` flag in the task commands we can noticeably decrease the time it takes to execute the tests

We can also reduce the amount of text being printed out by the `deno task coverage` command by asking Deno to set the the test reporter to dot with the `--reporter=dot` flag  

### testing

Clone this branch and run the `deno task test/converage` commands

### Requirements 

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/deno-slack-sdk/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've ran `deno task test` after making the changes.

### CODE REVIEW

File: deno.jsonc [hunk 1]
```diff
@@ -26,8 +26,8 @@
     }
   },
   "tasks": {
-    "test": "deno fmt --check && deno lint && deno bundle src/mod.ts && deno test --allow-read",
-    "coverage": "deno test --allow-read --coverage=.coverage && deno coverage --exclude=fixtures --exclude=test --lcov --output=lcov.info .coverage && deno run --allow-read https://deno.land/x/code_coverage@0.2.0/cli.ts"
+    "test": "deno fmt --check && deno lint && deno bundle src/mod.ts && deno test --allow-read --parallel",
+    "coverage": "deno test --reporter=dot --parallel --allow-read --coverage=.coverage && deno coverage --exclude=fixtures --exclude=test --lcov --output=lcov.info .coverage && deno run --allow-read https://deno.land/x/code_coverage@0.2.0/cli.ts"
```
Comment (filmaj) [inline deno.jsonc:30]: Do we need the reporter change here? The `deno run code_coverage` invocation at the end handles reporting.

**Review by filmaj — CHANGES_REQUESTED** [review]
Left a question re: `reporter` flag

File: deno.jsonc (hunk 1 above)
Comment (WilliamBergamin) [reply deno.jsonc:30]: Yes exactly the default report will print the test results but `--reporter=dot` will hide them as follows

`deno task coverage`

```txt
Task coverage rm -rf .coverage && deno test --reporter=dot --parallel --allow-read --coverage=.coverage src/  && deno coverage --exclude=fixtures --exclude=test --lcov --output=lcov.info .coverage && deno run --allow-read https://deno.land/x/code_coverage@0.2.0/cli.ts
..................................................................................................................................
..................................................................................................................................
..............................
ok | 176 passed (114 steps) | 0 failed (1s)

.------------------------------------------------------------------------------------------------------------------------------------------------.
|                              File Path                              | Coverage |                    Lines Without Coverage                     |
|---------------------------------------------------------------------|----------|---------------------------------------------------------------|
```

Comment (filmaj) [reply deno.jsonc:30]: Ohhhh that's awesome. I like it!

**Pull request merged**